src/change.js

```diff
@@ -1,11 +1,17 @@
 export default function isComplete(i, listArray, isComp, descript) {
   if (isComp.checked) {
-      if (listArray[i].completed === true) {
-        listArray[i].completed = false;
-        descript.style.textDecoration = 'none';
-      } else {
-        listArray[i].completed = true;
-        descript.style.textDecoration = 'line-through';    
-      }
+    if (listArray[i].completed === true) {
+      listArray[i].completed = false;
+      descript.style.textDecoration = 'none';
+    } else {
+      listArray[i].completed = true;
+      descript.style.textDecoration = 'line-through';
+    }
+  } else if (listArray[i].completed === true) {
+    listArray[i].completed = false;
+    descript.style.textDecoration = 'none';
+  } else {
+    listArray[i].completed = true;
+    descript.style.textDecoration = 'line-through';
```

Comment:
- [ ] Deleting a task function is missing please work on that

Reply:
the deleting function is not missing, it's in the task.js file, and it's working